lib/index.js

```diff
@@ -1,85 +1,70 @@
-/**
```

Comment:
the original license does not allow this to be removed.

Reply:
I pulled it out since I didn't see it in https://github.com/sindresorhus/find-up-simple/blob/main/index.js. I moved it to LICENSE to match our other repos and updated it to include our name since I believe that's the correct behavior when forking. see: https://github.com/orgs/community/discussions/47161#

Reply:
that's not adequate - if you're going to yank it out you still need to include his copyright notice. we can add ours, but we have to include his. the start of sindre's license:


MIT License

Copyright (c) Sindre Sorhus <sindresorhus@gmail.com> (https://sindresorhus.com)

Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:

The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.

Reply:
ok, i guess i was misinterpreting that answer I linked to. gonna put back the old text and include both names.